app/controllers/general.php

```diff
@@ -288,6 +288,9 @@ function router(App $utopia, Database $dbForConsole, callable $getProjectDB, Swo
             $execution->setAttribute('logs', $executionResponse['logs']);
             $execution->setAttribute('errors', $executionResponse['errors']);
             $execution->setAttribute('duration', $executionResponse['duration']);
+
+        } catch (AppwriteException $ex) {
+            throw $ex;
```

Comment:
We need to set some execution params here, like status, errors, body, ..
We also then need to save execution (createDocument) before we thorw.

Reply:
Done 👍